spec/lib/tasks/user_spec.rb

```diff
@@ -87,4 +88,24 @@
       @rake[task_name].invoke
     end
   end
+
+  describe 'usasearch:user:warn_set_to_not_approved' do
```

Comment:
wouldn't be a bad idea to include a test to make sure the warning is not sent prematurely. Maybe a similar test with a date of 75 days ago and using `expect(...)not.to`